packages/fhir-client/src/components/PatientListView/tests/index.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { MemoryRouter } from 'react-router-dom';
+import { PatientListView } from '..';
+
+describe('PatientListView', () => {
+  let queryClient;
+
+  beforeEach(() => {
+    queryClient = new QueryClient();
+  });
+
+  afterEach(() => {
+    queryClient.clear();
+  });
+
+  it('renders loading state when patientId is provided', async () => {
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <PatientListView fhirBaseURL="https://example.com/fhir" />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+
+    expect(screen.getByText(/loading/i)).toBeInTheDocument();
+    await waitFor(() => screen.getByText(/inactive/i)); // wait for the status to render
+  });
```

Comment:
Looks like a test case that could be exist as a simple check in a more elaborate test case like the ones below,